srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -2773,7 +2822,7 @@ CONFIG_THUNDER_NIC_PF=m
 CONFIG_THUNDER_NIC_VF=m
 CONFIG_THUNDER_NIC_BGX=m
 CONFIG_THUNDER_NIC_RGX=m
-CONFIG_CAVIUM_PTP=m
+CONFIG_CAVIUM_PTP=y
```

Comment:
why?

Reply:
This was "default y" in older Kernels. should be changed.